app/controllers/application_controller.rb

```diff
@@ -18,6 +18,10 @@ def configure_permitted_parameters
     end
   end
 
+  def default_url_options
+    { locale: I18n.locale }
+  end
+
   def set_locale
     I18n.locale = params[:locale] || I18n.default_locale
```

Comment:
Do we set `default_locale`?

Reply:
We don't need to unless we want to be explicit. It defaults to `en`.

From: "The I18n library will use English as a default locale, i.e. if a different locale is not set, :en will be used for looking up translations."